src/xpk/core/blueprint/blueprint_generator.py

```diff
@@ -354,6 +360,289 @@ def _get_a3_mega_blueprint_dependencies(
+                    "gpu_driver_version": "LATEST"
+                },
+            }],
+            "additional_networks": (
+                f"$(concat([{{network={cluster_name}-a3u-net-1.network_name,"
+                f" subnetwork={cluster_name}-a3u-net-1.subnetwork_name,"
+                f' subnetwork_project="{project_id}", nic_type="GVNIC",'
+                " queue_count=null, network_ip=null, stack_type=null,"
+                " access_config=[{nat_ip=null, public_ptr_domain_name=null,"
+                " network_tier=null}], ipv6_access_config=[],"
+                " alias_ip_range=[]}],"
+                f" {cluster_name}-a3u-rdma-net.subnetwork_interfaces_gke))"
+            ),
+        },
+        outputs=["instructions"],
+    )
+    if extended_reservation is not None:
+      gpu_pool.settings["reservation_affinity"] = {
+          "consume_reservation_type": "SPECIFIC_RESERVATION",
+          "specific_reservations": [{"name": extended_reservation}],
+      }
+    tas_install_id = "topology-aware-scheduler-install"
+    tas_install = DeploymentModule(
+        id=tas_install_id,
+        source="github.com/GoogleCloudPlatform/cluster-toolkit.git//community/modules/compute/gke-topology-scheduler?ref=e0c690b",
+        use=[cluster_id],
+    )
+    workload_manager_install_id = "workload-manager-install"
+    workload_manager_install = DeploymentModule(
+        id=workload_manager_install_id,
+        source="github.com/GoogleCloudPlatform/cluster-toolkit.git//modules/management/kubectl-apply?ref=e0c690b",
+        use=[cluster_id],
+        settings={
+            "kueue": {"install": True, "version": "v0.9.1"},
+            "jobset": {"install": True, "version": "v0.7.1"},
+            "apply_manifests": [
+                {"source": nccl_installer_path},
+                {"source": mglru_disable_path},
+            ],
+        },
+    )
+    job_template_id = "job-template"
+    job_template = DeploymentModule(
+        id=job_template_id,
+        source="modules/compute/gke-job-template",
+        use=[gpu_pool_id],
+        settings={
+            "image": "nvidia/cuda:11.0.3-runtime-ubuntu20.04",
+            "command": ["nvidia-smi"],
+            "node_count": 2,
+            "name": "run-nvidia-smi",
+        },
+        outputs=["instructions"],
+    )
+
+    primary_group = DeploymentGroup(
+        group="primary",
+        modules=[
+            gpu_net_0,
+            gpu_net_1,
+            rma_net,
+            a3_ultra_cluster,
+            gpu_pool,
+            tas_install,
+            workload_manager_install,
+            job_template,
+        ],
+    )
+    a3_ultra_blueprint = Blueprint(
+        blueprint_name=blueprint_name,
+        deployment_groups=[primary_group],
+        vars={
+            "project_id": project_id,
+            "deployment_name": blueprint_name,
+            "region": region,
+            "zone": zone,
+        },
+        terraform_providers={
```

Comment:
Apparently, we are going to have these versions in next version of CT by default? we can remove them when we point to main (if happens today).
https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/ed822a5b3f8136f4e2924fb1d3300cbe90dc7ddc/pkg/config/expand.go#L199

Reply:
we can remove these terraform providers, the version "6.13.0" is now set by default in CT's new release
https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/6a19416f7607ea95adb5e461efee27132868fbf9/pkg/config/expand.go#L199